pkg/gui/gui.go

```diff
@@ -110,6 +110,10 @@ type Gui struct {
 	// lazygit was opened in, or if we'll retain the one we're currently in.
 	RetainOriginalDir bool
 
+	refOperations            map[string]types.RefOperation
```

Comment:
Would be good to add a comment here explaining when it's appropriate to set a ref operation. At the moment the rule is to use a ref operation when we need to talk to the remote

Reply:
5b64b2bface7